Assignment2/main.cpp

```diff
@@ -0,0 +1,89 @@
+#include <iostream>
+#include <fstream>
+
+
+
+
+int main() {
+
+    std::string menu = "";
+    std::cout << "Enter init or edit" << std::endl;
+    std::cin >> menu;
+    std::cout << "Menu selected" << std::endl;
+    if (menu == "edit") {
```

Comment:
This section is a good start! Here is what you need to consider to complete it. How can you know what is in the file so you only have to ask the user to supply one input for the property/field named by `argv[2]`

the invocation would be `gen.exe edit email` for example. So the program will only ask for the email input but will have to maintain the other values in the file.
HINT if I just told you to recreate the file from the contents of the file. Like copying the file what could you do procedurally to make that happen.